web/game/game_state.ex

```diff
@@ -26,20 +26,30 @@ defmodule ElMascarar.GameState do
       raise "CannotSwitchOwnCard"
     else
       game = ready(game)
-      allPreviousCards = game.players ++ game.free_cards
-      myPreviousCard = Enum.at(game.players, active_player_card_number)
-      theirPreviousCard = Enum.at(allPreviousCards, card_number)
+      all_previous_cards = game.players ++ game.free_cards
+      my_previous_card = Enum.at(game.players, active_player_card_number)
+      their_previous_card = Enum.at(all_previous_cards, card_number)
       if switch do
-        myCard = myPreviousCard |> Map.put(:true_card, theirPreviousCard.true_card) |> Map.put(:card, "SwitchedOrNot")
-        theirCard = theirPreviousCard |> Map.put(:true_card, myPreviousCard.true_card) |> Map.put(:card, "SwitchedOrNot")
+        my_card =
+          my_previous_card |>
+          Map.put(:true_card, their_previous_card.true_card) |>
+          Map.put(:card, "SwitchedOrNot")
```

Comment:
You can replace these two lines with this:
```elixir
Map.merge(%{true_card: their_previous_card.true_card, card: "SwitchedOrNot"})
```
http://elixir-lang.org/docs/stable/elixir/Map.html#merge/2